core/src/components/dropdown/dropdown.stories.tsx

```diff
@@ -177,6 +177,7 @@ const defaultOptionLookUp = {
   'Option 1': 'option-1',
   'Option 2': 'option-2',
   'Option 3': 'option-3',
+  'Option 4': 'option-4',
```

Comment:
When multiselect is active and option types are rendered via the options prop there is an option to put "Option 4" as a default option, but it doesn't exist in the Dropdown itself. Might be an edge case though so not sure it's worth addressing at this point?
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/25a2583b-9568-4ac4-85e8-16948b0d51ee)